unity-renderer/Assets/DCLPlugins/ECS7/InternalECSComponents/Interfaces/InternalECSComponentModels.cs

```diff
@@ -71,6 +70,22 @@ public class InternalRenderers : InternalComponent
         public IList<Renderer> renderers = new List<Renderer>();
     }
 
+    public class InternalAudioSource : InternalComponent
+    {
+        public AudioSource audioSource;
+    }
+
+    public class InternalSceneBoundsCheck : InternalComponent
+    {
+        public Vector3 entityPosition = Vector3.zero;
+        public Bounds entityLocalMeshBounds = new Bounds();
+        public bool meshesDirty = false;
+        public IList<Renderer> renderers;
+        public IList<Collider> physicsColliders;
+        public IList<Collider> pointerColliders;
+        public AudioSource audioSource;
```

Comment:
Why do we need to have `AudioSource` here when we have `InternalAudioSource`?

Reply:
looks like it's also there to just iterate through `InternalSceneBoundsCheck` component to set those reference in or out of scene boundaries instead of asking for the `InternalAudioSource` component to perform any modification.
Please keep in mind this is the first of probably many more iterations for this system, I can totally see us replacing `InternalAudioSource` for something like `InternalMedia` to handle all media components in a single one instead of just the `AudioSource` component